bin/lov_migrator

```diff
@@ -0,0 +1,238 @@
+#!/usr/bin/env ruby
+
+# Exit cleanly from an early interrupt
+Signal.trap("INT") { exit 1 }
+
+require 'optparse'
+
+options = {}
+OptionParser.new do |opts|
+  opts.banner = "Usage: lov_migrator [options]"
+  opts.on( '-a', '--all') do 
+    options[:vocabs] = [:all]
+  end
+  opts.on('-v', '--vocabularies PREFIX1, PREFIX2', 'Comma-separated list of vocabularies to update or import') do |acronym|
+    ontologies_acronyms = acronym
```

Comment:
why aren't u initializing options here with something like:
`options[:acronyms] = acronym.split(',')`

I don't understand the utility of this var `ontologies_acronyms`

Reply:
So not used yet, but someday maybe